Mathlib/CategoryTheory/Conj.lean

```diff
@@ -15,13 +16,12 @@ An isomorphism `α : X ≅ Y` defines
 - a group isomorphism `CategoryTheory.Iso.conjAut : Aut X ≃* Aut Y` by
   `α.conjAut f = α.symm ≪≫ f ≪≫ α`.
 
-For completeness, we also define
-`CategoryTheory.Iso.homCongr : (X ≅ X₁) → (Y ≅ Y₁) → (X ⟶ Y) ≃ (X₁ ⟶ Y₁)`,
-cf. `Equiv.arrowCongr`,
-and `CategoryTheory.Iso.isoCongr : (f : X₁ ≅ X₂) → (g : Y₁ ≅ Y₂) → (X₁ ≅ Y₁) ≃ (X₂ ≅ Y₂)`.
+using
+`CategoryTheory.Iso.homCongr : (X ≅ X₁) → (Y ≅ Y₁) → (X ⟶ Y) ≃ (X₁ ⟶ Y₁)`
+and `CategoryTheory.Iso.isoCongr : (f : X₁ ≅ X₂) → (g : Y₁ ≅ Y₂) → (X₁ ≅ Y₁) ≃ (X₂ ≅ Y₂)`
+which are defined in  `CategoryTheory.HomCongr`.
```

Comment:
Something seems to have gone wrong here: sentence begins with "using"?

Reply:
It's all one sentence, starting with "An isomorphism". I was hoping this comment would be helpful for folks familiar with the old file "conj".